modules/ROOT/pages/support-center-access.adoc

```diff
@@ -6,10 +6,17 @@ endif::[]
 
 image::support-logo.png[support]
 
-Your account currently doesn't have access to the Support Center. This may be because your account is a free account, a trial, or has not been fully set up according to your order. If you are a customer and you believe you have reached this page by mistake and should have access to the Support Center, please mailto:info@mulesoft.com[contact us].
+You are seeing this page because your account doesn't currently have access to the MuleSoft Support Center. This could be because your account is a free account, a trial, or is not fully set up according to your order.
 
-You always have access to all the public resources available at our http://support.mulesoft.com[Support Center].
+Verify that you are using the correct credentials to sign in. 
+
+* Anypoint Platform users sign in using their Anypoint Platform user name and password. 
+* On premises users use the credentials provided by your MuleSoft Customer Success Manager.
+
+If you have an existing commercial relationship with MuleSoft and are experiencing problems accessing your Anypoint Platform account, please raise a case from: https://support.mulesoft.com[MuleSoft Support Center]. If you can't access the Support portal, please contact your Customer Success Manager or MuleSoft Account Management team for assistance.
+
+You always have access to the public resources available at our http://support.mulesoft.com[MuleSoft Support Center].
```

Comment:
Saying that users have access sounds like you're saying they can log in for free. How about changing this to: "You can view public information at the MuleSoft Support site."